### DISCUSSION TITLE
Is this the real bleach bit?

### DISCUSSION THREAD

**Discussion by AverageUser177** [bleachbit/bleachbit#1525] [Q&A]
I found this on the Microsoft store but I’m not sure if it’s the actual legit owners of bleachbit is this official? If it is I would be willing to support it by buying it. https://apps.microsoft.com/store/detail/bleachbit-system-cleaner/9N9J50V080PS

**Comment by az0** [comment]
> it’s the actual legit owners of bleachbit is this official?

Basically, no: The publisher _Brian studio_ is not affiliated with this project.  I can't tell you what changes were made before it was put on the Windows Store, so for security reasons, check that the application and installer are [digitally signed](https://docs.bleachbit.org/doc/install-on-windows.html) by Andrew Ziem.

See also this issue: https://github.com/bleachbit/bleachbit/issues/1346

>If it is I would be willing to support it by buying it

That is kind of you. For direct ways to help, please see the [contribute](https://www.bleachbit.org//contribute) page.
